storybook/src/example-pages/toepassingen/loket/homepage.ts

```diff
@@ -13,9 +13,13 @@ examplePageFactory(
     { anchorTemplate, linkListTemplate, highlightBoxTemplate, richContentTemplate, tileTemplate, buttonTemplate },
     templates,
   ) => html`
+    <dso-skiplink label="Ga naar navigatie" to="navigation"></dso-skiplink>
```

Comment:
Deze templates niet uitschrijven maar gebruik maken van `skiplinkTemplate()`. Die is beschikbaar bij de templates op regel 13. Daar kan die worden toegevoegd en vervolgens met een template literal worden aangeroepen:

```html
${skiplinkTemplate({ label: "Ga naar navigatie", to: "navigation" })
```

Reply:
dit lukt me niet 1-2-3. straks nog even naar kijken

Reply:
Hoe moet ik dat exact doen? alles wat ik probeer resulteert in een berg foutmeldingen...

Reply:
@iterox kan jij deze nog verwerken?

Reply:
Gedaan, met die kanttekening dat in de `skiplink.core-template` 

```tsx
if (!e.detail.isModifiedEvent) {
            e.detail.originalEvent.preventDefault();
          }
          ```
          
          ingrijpt en bijv. de skiplink naar de footer dusn iet naar de footer gaat. Wat wel werkt als we niet de template gebruiken , maar direct het element

Reply:
@iterox je bedoelt zoals ik het had gedaan?

Reply:
Zie de laatste commit. Ik heb de preventDefault() afhandeling verplaatst naar Args omdat we dit alleen binnen de context van die demo pagina willen hebben. Hierdoor scrollt het op de (fullscreen!) voorbeeldpagina van Loket/Homepage.